utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -32,7 +31,6 @@
 "Apache License Version 2.0": Apache-2.0
 "Apache License v2": Apache-2.0
 "Apache License v2.0": Apache-2.0
-"Apache License": Apache-2.0
```

Comment:
Apache license is 99,99% of the cases Apache 2.0 as previous version of this licenses (Apache-1.0, Apache-1.1) are super rare. I was thinking maybe we could implement a mapping for aboguous licenses based on year. Apache-2.0 was released in 2004 so could we say "if a project is from 2004 or younger map Apache License to Apache-2.0" Thoughts?

Reply:
Even taking the year into account this way does not give 100% confidence. And we currently do not capture a project's / package's release date, which would be a significant change (but a good one anyway; I'd also like to be able to warn users about old dependencies in their tree that haven't seen a new release for years).